ui/src/app/components/copy-modal.tsx

```diff
@@ -161,25 +211,29 @@ class CopyModal extends React.Component<Props, State> {
     }
   }
 
+  setDestination(destination: Workspace) {
```

Comment:
this method doesn't have much to do yet but it will in the next PR

Reply:
two lines is plenty of work for a function to do